.github/workflows/ci.yml

```
@@ -0,0 +1,54 @@
+name: CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  test:
+    name: rake test
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+        ruby-version: ['3.1', '3.0', '2.7']
+        libexif-version: ['system', 'latest']
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install libexif from system
+        run: sudo apt-get install libexif-dev
+        if: ${{ matrix.libexif-version == 'system' }}
+      - name: Uninstall libexif from system
+        run: sudo apt-get remove libexif12 libexif-dev
+        if: ${{ matrix.libexif-version == 'latest' }}
+      - uses: robinraju/release-downloader@v1.3
+        name: Download latest libexif
+        if: ${{ matrix.libexif-version == 'latest' }}
+        with:
+          repository: "libexif/libexif"
+          latest: true
+          zipBall: false
+          tarBall: true
+      - name: Install latest libexif
+        if: ${{ matrix.libexif-version == 'latest' }}
+        run: |
+          sudo apt-get install autopoint
+          mkdir libexif
+          tar xvf libexif-libexif*.tar.gz -C libexif --strip-components 1
+          cd libexif
+          autoreconf -i
+          ./configure --prefix=/usr
+          make
+          sudo make install
```

```suggestion
      - name: Checkout libexif
        uses: actions/checkout@v2
        with:
          repository: libexif/libexif
          ref: ${{matrix.libexif-version}}
          path: libexif
      - name: Set up libexif
        run: |
          sudo apt-get update
          sudo apt-get remove libexif12 libexif-dev
          sudo apt-get install autopoint
          cd libexif
          autoreconf -i
          ./configure --disable-docs --prefix=/usr
          make -j $(nproc)
          sudo make install
```

I think we should use `actions/checkout@v2` instead of `robinraju/release-downloader@v1.3` because it's maintained by GitHub and simple is good.